frame/babe/src/benchmarking.rs

```diff
@@ -17,8 +17,6 @@
 
 //! Benchmarks for the BABE Pallet.
 
-#![cfg_attr(not(feature = "std"), no_std)]
```

Comment:
We already have this in `lib.rs` hence why it's unneeded. I probably just copy-pasted it when writing the benchmark.